nano/node/ipc.cpp

```diff
@@ -407,6 +407,7 @@ void nano::ipc::ipc_server::stop ()
 	{
 		transport->stop ();
 	}
+	node.logger.always_log ("IPC: server stopped");
```

Comment:
Can we also transports.clear () here so multiple calls to stop won't have an effect?